assets/xml/objects/object_kin2_obj.xml

```diff
@@ -1,34 +1,34 @@
 ﻿<Root>
     <File Name="object_kin2_obj" Segment="6">
         <DList Name="object_kin2_obj_DL_000120" Offset="0x120" />
-        <DList Name="object_kin2_obj_DL_000128" Offset="0x128" />
-        <DList Name="object_kin2_obj_DL_0002C0" Offset="0x2C0" />
-        <DList Name="object_kin2_obj_DL_000360" Offset="0x360" />
-        <Collision Name="object_kin2_obj_Colheader_000490" Offset="0x490" />
+        <DList Name="gOceanSpiderHouseBombableWallDebrisDL" Offset="0x128" />
+        <DList Name="gOceanSpiderHouseBombableWallCrackDL" Offset="0x2C0" />
+        <DList Name="gOceanSpiderHouseBombableWallDL" Offset="0x360" />
+        <Collision Name="gOceanSpiderHouseBombableWallColHeader" Offset="0x490" />
```

Comment:
When naming `Collision`s convention is to just suffix with `Col`

Reply:
Adjusted in next commit to be pushed.